index.html

```diff
@@ -5610,21 +5613,31 @@ <h3>Language and Base Direction</h3>
 
         <pre class="example nohighlight" title="Design pattern for natural language strings">
 "<a>property</a>": {
-  "value": "<span class="highlight">The string value</span>",
-  "lang": "<code>LANGUAGE</code>"
-  "dir": "<code>DIRECTION</code>"
+  "@value": "<span class="highlight">The string value</span>",
+  "@language": "<code>LANGUAGE</code>"
+  "@direction": "<code>DIRECTION</code>"
 }
         </pre>
 
+        <p>
+When the language value object is used in place of a string value, the object
+MUST contain a `@value` property whose value is a string, and SHOULD contain a
+`@language` property whose value is a string containing a well-formed
```

Comment:
The change below applies @aphillips request:

```suggestion
`@language` property whose value is a string containing a valid
```

We are trying very hard to stay away from any usage of "well-formed" text in the spec.

Reply:
"well-formed" and "valid" are defined terms in BCP47 (see [here](https://www.rfc-editor.org/rfc/rfc5646.html#section-2.2.9)). They are also defined terms in the i18n glossary. Generally VC shouldn't require implementations to check for validity, because that involves reading the IANA Language Subtag Registry--you actually want conformance to hinge on well-formed language tags. But you wan users to pick language tags that are valid.

In this case, though, I can agree with the change provided you're aware of what you want to require people to demonstrate.

Reply:
k. Putting this back to "well-formed".